### DISCUSSION TITLE
Redash Dashboard Access Control

### DISCUSSION THREAD

**Discussion by gopal-faircent** [getredash/redash#6341] [Ideas]
Hi team, First of thank you team for having a such great tool.
Currently we have only option to control access restriction using data source. 
Bit, I have request can we create Dashboard Access Control basis on what Dashboard access basis on user/group.
Example: Dashboard A access restricted to user1 only.; Dashboard B access restricted to user 2 & 3 only. So on..
and for Query > we can delegate the permission to other user to edit query.

Awaiting for your response.

**Comment by ayush-421** [comment]
Hi!

This can be done using a hackish solution as of now. I wanted to do the same thing and what worked for me is, I created a duplicate of my data source and created a new dashboard using this data source and granted limit access to this data source. So basically I put the access control on the data source associated to the dashboard which kind of works the same way. But it would be nice if this is given as a feature altogether.

**Comment by gopal-faircent** [comment]
This solution will not work as multiple Dashboard access need to provide multiple users or wise-versa.

Just for information, like in Metabase they have solution with name of Collection where we can create a collection group add the Dashboard or Query and provide the permission to specific user.

**Comment by konnectr** [comment]
Hi. I think this is a sore subject for Redash
https://github.com/getredash/redash/issues/3284 .
All I can offer is to participate in the discussion in issue.

**Comment by gopal-faircent** [comment]
Okay, Is this any planning to implement the revamp of permission structure
with access control.

Do Redash team have any ETA?

**Reply by justinclift** [reply]
We've recently moved to being a "Community Led" project, so we've been mostly cleaning up the source code, merging PR's, and so on.  Once that's done we'll probably start doing proper planning of longer term direction / things we want fixed /etc.

Not likely to be happening within the next few weeks. :wink:
